### PR TITLE
Update ko_kr.json for 0.4

### PR DESCRIPTION
**Pull request by Kardane**
Hello, I heard you've got updates! Here's my KR translation file.

### CODE REVIEW

File: src/main/resources/assets/farmersdelight/lang/ko_kr.json [hunk 1]
```diff
@@ -67,22 +72,24 @@
...
   "item.farmersdelight.ham": "햄",
   "item.farmersdelight.smoked_ham": "훈제 햄",
-
+  
   "item.farmersdelight.pie_crust": "파이 껍질",
   "item.farmersdelight.cake_slice": "케이크 조각",
   "item.farmersdelight.apple_pie_slice": "사과 파이 조각",
   "item.farmersdelight.sweet_berry_cheesecake_slice": "달콤한 열매 치즈케이크 조각",
   "item.farmersdelight.chocolate_pie_slice": "초콜릿 파이 조각",
-  "item.farmersdelight.fruit_salad": "과일 샐러드",
```
Comment (vectorwing) [inline src/main/resources/assets/farmersdelight/lang/ko_kr.json]: From Google Translate, it seems the previous would be correct, with the new one being just "fruit". Is this accurate?

File: src/main/resources/assets/farmersdelight/lang/ko_kr.json [hunk 2]
```diff
@@ -113,28 +122,35 @@
   "item.farmersdelight.fried_rice": "볶음밥",
   "item.farmersdelight.pumpkin_soup": "호박 수프",
   "item.farmersdelight.baked_cod_stew": "대구 스튜",
+  "item.farmersdelight.noodle_soup": "우동",
 
+  "item.farmersdelight.honey_glazed_ham": "꿀 바른 돼지고기 구이",
   "item.farmersdelight.pasta_with_meatballs": "미트볼 파스타",
   "item.farmersdelight.pasta_with_mutton_chop": "양고기 파스타",
+  "item.farmersdelight.roasted_mutton_chops": "구운 양고기 요리"
```
Comment (vectorwing) [inline src/main/resources/assets/farmersdelight/lang/ko_kr.json]: You forgot a comma on this line (see red-highlight colon below).

**Review by vectorwing — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/farmersdelight/lang/ko_kr.json (hunk 1 above)
Comment (Kardane) [reply src/main/resources/assets/farmersdelight/lang/ko_kr.json]: Oof. I accidentally deleted salad part 😅

File: src/main/resources/assets/farmersdelight/lang/ko_kr.json (hunk 2 above)
Comment (Kardane) [reply src/main/resources/assets/farmersdelight/lang/ko_kr.json]: Just added comma, Thx!

**Review by vectorwing — APPROVED** [review]

**Comment by vectorwing** [comment]
For some reason, GitHub is treating the changes as conflicts... Not sure why, though, since it's conflicting from 1.16.3.

**Comment by vectorwing** [comment]
OK, seems I was able to resolve it manually! 😄
Thank you for the update!

**Pull request merged**